index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/3866333966.js" crossorigin="anonymous"></script>
+    <title>Document</title>
+</head>
+<body>
+    <!--THE NAVIGATION BAR SECTION-->
+    <aside class="menu-bar">
+        <i class="fa fa-close fa-2x hamburger white"></i>
+        <ul class="menu-bar-ul">
+            <li class="mobile-list"><a href="about.html" class="white">About</a></li>
+            <li class="mobile-list"><a href="#" class="white">Program</a></li>
```

Comment:
- [ ] On the mobile menu there is some gap above the navbar. Kindly try to fix this issue.

Your
![image](https://user-images.githubusercontent.com/53019916/127886838-2be39d00-514b-41ed-afa7-23575b17f24e.png)